packages/python/plotly/plotly/express/_core.py

```diff
@@ -1576,6 +1577,9 @@ def aggfunc_continuous(x):
     if args["color"]:
         if not args["hover_data"]:
             args["hover_data"] = [args["color"]]
+        elif isinstance(args["hover_data"], dict):
+            if not args["hover_data"].get(args["color"]):
+                args["hover_data"][args["color"]] = (True, None)
```

Comment:
in a first version I had put only `True` this is why I needed to modify the code later on but by putting a tuple directly there's no need for this anymore.

Reply:
ok great!